README.md

```diff
@@ -49,16 +49,26 @@ snowdrop	    https://snowdrop.github.io/helm
 ## Usage
 
 Requirements:
-- Connected/logged to a Openshift cluster
+- Connected/logged to a Kubernetes/Openshift cluster
 - Have installed [the Helm command line](https://helm.sh/docs/intro/install/)
 
 ### spring-boot-example-app
 
-This chart deploys and exposes a Spring Boot application in Openshift.
-To install it, you need to execute the following command:
+This chart deploys and exposes a Spring Boot application in Kubernetes or Openshift.
+
+- For Kubernetes:
+To use it in Kubernetes, the image of the Spring Boot application needs to be available in a public container registry. In this example, we'll use the image `quay.io/user/my-app:latest`. To install it, you need to execute the following command:
```

Comment:
`...it in kubrnetes...` --> `on kubernetes` 
To be changed everywhere !

and

`to be available in a public image registry` --> `to be published on an images registry where you have access and be logged`